test/module/iroha-cli/client_test.cpp

```diff
@@ -188,6 +195,48 @@ TEST_F(ClientServerTest, SendTxWhenStatelessInvalid) {
   ASSERT_NE(res.answer.error_message().size(), 0);
 }
 
+TEST_F(ClientServerTest, SendTxWhenStatefulInvalid) {
```

Comment:
gwt doc is missing, same for the following

Reply:
All tests in this file are missing the docs, should I still write it for this test?

Reply:
Honestly that test require significant amount of refactoring/reworking. That doc comment may be really helpful, so yes it better be written